usr.sbin/inetd/inetd.h

```diff
@@ -0,0 +1,128 @@
+#ifndef _INETD_H
+#define _INETD_H
+
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <sys/time.h>
+#include <sys/un.h>
+
+#include <arpa/inet.h>
+
+#include <netdb.h>
+
+#include "pathnames.h"
+
+#ifdef IPSEC
+#include <netipsec/ipsec.h>
+#ifndef IPSEC_POLICY_IPSEC	/* no ipsec support on old ipsec */
+#undef IPSEC
+#endif
+#include "ipsec.h"
+#endif
+
+#define NORM_TYPE	0
```

Comment:
could also use an enum

Reply:
This was just existing inetd code. It's probably somewhat simple to update if you think this would be a good opportunity to do that.

Reply:
I think modernizing the code is good, let's do it.